src/client_impl.cc

```diff
@@ -42,19 +42,19 @@ MixerClientImpl::MixerClientImpl(const MixerClientOptions &options)
 
 MixerClientImpl::~MixerClientImpl() { check_cache_->FlushAll(); }
 
-void MixerClientImpl::Check(Attributes &attributes, DoneFunc on_done) {
+void MixerClientImpl::Check(const Attributes &attributes, DoneFunc on_done) {
   auto response = new CheckResponse;
   Status status = check_cache_->Check(attributes, response);
```

Comment:
it will be faster if Check() can return signature.  Not need to calculate signature again for cache miss